client/src/app/app.component.scss

```diff
@@ -0,0 +1,37 @@
+// Standard Styles
+body {
+    /* Margin bottom by footer height */
+    margin-bottom: 60px;
```

Comment:
The comment here seems to convey that this offset is being applied to account for the `footer` occupying `60px` at the bottom of the screen. The `footer` is a child of the `main` element, which has the `app__main` class, so why not define this rule in the `app__main` class?